rasa/core/policies/ted_policy.py

```diff
@@ -476,6 +482,10 @@ def load(cls, path: Union[Text, Path], **kwargs: Any) -> "TEDPolicy":
         )
         meta = train_utils.update_similarity_type(meta)
 
+        meta["should_finetune"] = should_finetune
+        if "epoch_override" in kwargs:
+            meta[EPOCHS] = kwargs["epoch_override"]
```

Comment:
Do we also have this for DIET?

Reply:
It happens but slightly differently. The difference in implementation is because Policies do not accept something similar to `RasaNLUModelConfig`. For DIET, we directly change number of epochs in the config and then pass to `load` methods of the components.